src/content/docs/build/env-variables/add-manage-env-variables.mdx

```diff
@@ -0,0 +1,40 @@
+---
+page_id: 82646df5-5e4c-4b09-babf-2f77c36bd73b
+title: Add and update environment variables
+sidebar:
+  order: 2
+relatedArticles:
+  - 2d39a390-baf8-4f49-a18c-950d3f248c65
+  - d95a8edd-f61a-4095-a80e-ffd67f90a438
+---
+
+Environment variables are for including sensitive information in your application code, using a key. You then update the variable's key-value to update the value wherever the key is referenced. You can store as many environment variables in Kinde as you want. 
```

Comment:
@clairekinde11 I'd be wary of using the term `sensitive` here as we do have a type of env variable which can be marked as sensitive.

Environment variables are key-value pairs configured outside your source code so that each value can change depending on the Environment (link to env). Common use cases are for API keys, URLs or IDs which change per environment.

Reply:
Updated, please check